_includes/shared/markup/ccloud/config-create.adoc

```diff
@@ -0,0 +1,9 @@
+Create a development configuration file at `configuration/dev.properties` with your Confluent Cloud information.
+This file and the secrets it contains can be easily created using the Confluent Cloud UI. To do so, navigate to the
+``Clients`` section in your cluster, click ``Create Kafka cluster API key & secret`` and ``Create Schema Registry API key & secret``,
+lastly select ``Java`` as the client. You should see something similar to below in the UI. Copy the contents of the configuration file and paste them into a
```

Comment:
Maybe tell the user `show API keys` must be checked , otherwise they don't render

Reply:
True that, added in commit https://github.com/confluentinc/kafka-tutorials/pull/786/commits/71cceceb46a091134d731263f01929e75b9e687c